### ISSUE TITLE
Dropdown: searchInput onChange fires Dropdown onChange as well

### ISSUE THREAD

**Issue by mic4ael** [Semantic-Org/Semantic-UI-React#3122]
### Steps
1. Create a `Dropdown` component
2. Override its `searchInput` so that it provides a custom `onChange` callback.

### Expected Result
Typing inside of `Dropdown.SearchInput` should only trigger its `onChange`.

### Actual Result
Typing triggers input's `onChange` first and then dropdown's `onChange` callbacks.

### Version
0.82.3

### Testcase
https://codesandbox.io/s/40mx54ppr9

**Comment by layershifter** [comment]
Thanks, awesome report! PR is coming.

**Comment by mic4ael** [comment]
Amazing work! You guys rock :guitar:

**Comment by layershifter** [comment]
Fixed in #3123, will come with the next release.

**Comment by mic4ael** [comment]
Just a question, you have overrided `onSearchChange` in your test, whereas I mentioned that it's
`onChange` that causes the problem. Is that correct?

```
wrapperShallow(
    <Dropdown
        onSearchChange={onSearchChange}
        options={options}
        search
        searchInput={{ onChange: onInputChange }}
    />,
)
```
